lib/Monads/reader_option/Reader_Option_VCG.thy

```diff
@@ -83,6 +83,10 @@ lemma ofail_wp[wp]:
   "ovalid (\<lambda>_. True) ofail Q"
   by (simp add: ovalid_def ofail_def)
 
+lemma ask_wp:
```

Comment:
I'm never the best judge on whether a lemma should be in one of these named theorem sets like `[wp]`, especially in `Lib` rules, where they're sometimes added in particular spots. But I can certainly make this one `[wp]` if that's best

Reply:
This one would is basic enough to make sense as `[wp]`, yes. @corlewis any concerns there?